Index: deploy/Dockerfile
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>FROM ubuntu:latest\nLABEL authors=\"dameienik\"\n\nENTRYPOINT [\"top\", \"-b\"]
===================================================================
diff --git a/deploy/Dockerfile b/deploy/Dockerfile
--- a/deploy/Dockerfile	(revision 72a92c2a49e7190ad12eadc3f335fdd571021fca)
+++ b/deploy/Dockerfile	(date 1753382931529)
@@ -1,4 +1,15 @@
-FROM ubuntu:latest
-LABEL authors="dameienik"
+FROM golang:1.24.5-alpine3.22 AS builder
+
+COPY . /github.com/en7ka/auth/source
+WORKDIR /github.com/en7ka/auth/source
+
+RUN go mod download
+
+RUN go build -o ./bin/user-server ./cmd/grpc_server/main.go
+
+FROM alpine:latest
+
+WORKDIR /root/
+COPY --from=builder /github.com/en7ka/auth/source/bin/user-server .
 
-ENTRYPOINT ["top", "-b"]
\ No newline at end of file
+CMD ["./user-server"]
\ No newline at end of file
Index: cmd/grpc_server/main.go
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package grpc_server\n
===================================================================
diff --git a/cmd/grpc_server/main.go b/cmd/grpc_server/main.go
--- a/cmd/grpc_server/main.go	(revision 72a92c2a49e7190ad12eadc3f335fdd571021fca)
+++ b/cmd/grpc_server/main.go	(date 1753382961557)
@@ -1,1 +1,7 @@
-package grpc_server
+package main
+
+import "fmt"
+
+func main() {
+	fmt.Println("Hello World")
+}
